src/main/webapp/help/help-project.html

```diff
@@ -0,0 +1,3 @@
+<p>
+  Gtialb project: a String like namespace/project or the exactly project id (an integer like 1)
```

Comment:
typo: `Gtialb` > `Gitlab`

Reply:
Always GitLab if talking about the formal product name.